desktop/app.js

```diff
@@ -162,7 +162,7 @@ function createMenuTemplate( settings ) {
 			type: 'separator'
 		} );
 		fileMenu[ 'submenu' ].push( {
-			label: 'Exit',
+			label: '&Exit',
```

Comment:
@efegurkan it's been a long while since you did this work and I would love to help you get it merged into the project. Are you still interested in updating from @roundhill's suggestion above?